Cargo.toml

```diff
@@ -28,8 +28,8 @@ tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy"
   "stopwords",
   "lz4-compression",
 ], default-features = false }
-pgrx = "=0.14.3"
-pgrx-tests = "=0.14.3"
+pgrx = { git = "https://github.com/stuhood/pgrx.git", branch = "stuhood.if-let-for-option-integration" }
```

Comment:
what?  no.  we do not want to be working off of a pgrx fork

Reply:
Understood. I post these in this way to prove that the fix actually works: it's PR'd over here: https://github.com/pgcentralfoundation/pgrx/pull/2089 ... if we don't want to wait for the fix to land and then be released, then we'd apply a workaround instead. But skipping straight to the workaround doesn't integration test the fix.

Reply:
Fair.

We spent quite a bit of time using our own pgrx fork because the primary pgrx maintainer is, apparently, lazy.  It was a very annoying and confusing situation.  

I'll get a new pgrx out this weekend.

Reply:
> I'll get a new pgrx out this weekend.

The workaround is only one `allow(xxx)` statement, so no need to rush on this if that doesn't fit your schedule: just let me know!